.evergreen/config.yml

```diff
@@ -1528,6 +1556,37 @@ tasks:
       - func: run-aws-auth-test-with-aws-EC2-credentials
       - func: run-aws-ECS-auth-test
 
+  - name: "test-standalone-versioned-api"
+    tags: ["versioned-api"]
+    commands:
+      - func: bootstrap-mongo-orchestration
+        vars:
+          TOPOLOGY: "server"
+          AUTH: "auth"
+          SSL: "nossl"
+          REQUIRE_API_VERSION: true
+      - func: run-versioned-api-test
+        vars:
+          TOPOLOGY: "server"
+          AUTH: "auth"
+          SSL: "nossl"
+          REQUIRE_API_VERSION: true
+
+  - name: "test-standalone-versioned-api-test-commands"
```

Comment:
This new task runs tests in `./mongo`, `./mongo/integration` and `./mongo/integration/unified` against a standalone with `enableTestCommands=1`, `acceptAPIVersion2=1` and `requireAPIVersion=0`. Technically, this standalone config exists _just_ to run the new spec test `test-commands-deprecation-errors`, so running so many tests might be overkill. Let me know if so; I could make a new `Makefile` recipe that just runs the single test...